### PR TITLE
Reset token amount input field after execute

### PR DESCRIPTION
**Pull request by lazarmitic**
Issue: https://github.com/tempus-finance/tempus-app/issues/365

### CODE REVIEW

File: packages/tempus-client/src/components/detail/advanced/detailMint.tsx [hunk 1]
```diff
@@ -119,6 +119,10 @@ const DetailMint: FC<DetailMintProps> = props => {
     );
   }, [amount, backingToken, poolDataAdapter, selectedToken, tempusPool.address, userWalletAddress]);
 
+  const onExecuted = useCallback(() => {
+    setAmount('');
+  }, []);
```
Comment (danieledamiani) [inline packages/tempus-client/src/components/detail/advanced/detailMint.tsx:124]: `setAmount` in the dependency list

File: packages/tempus-client/src/components/detail/advanced/detailSwap.tsx [hunk 2]
```diff
@@ -141,6 +141,10 @@ const DetailSwap: FC<DetailSwapProps> = props => {
     );
   }, [poolDataAdapter, amount, tempusPool.ammAddress, tokenFrom.tokenAddress, tokenTo.tokenAddress, userWalletAddress]);
 
+  const onExecuted = useCallback(() => {
+    setAmount('');
+  }, []);
```
Comment (danieledamiani) [inline packages/tempus-client/src/components/detail/advanced/detailSwap.tsx:146]: fix dependencies

File: packages/tempus-client/src/components/detail/basic/detailDeposit.tsx [hunk 3]
```diff
@@ -149,6 +149,10 @@ const DetailDeposit: FC<PoolDetailProps> = ({ tempusPool, content, signer, userW
     }
   }, [signer, poolDataAdapter, ammAddress, backingToken, selectedToken, selectedYield, amount, minTYSRate]);
 
+  const onExecuted = useCallback(() => {
+    setAmount('');
+  }, []);
+
   const onApproveChange = useCallback(approved => {
     setTokensApproved(approved);
   }, []);
```
Comment (danieledamiani) [inline packages/tempus-client/src/components/detail/basic/detailDeposit.tsx:158]: fix dependencies

File: packages/tempus-client/src/components/detail/advanced/detailMint.tsx (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client/src/components/detail/advanced/detailMint.tsx:124]: Do we need to set it though? ESLint does not report it as an error, setAmount() never changes, it always does the same thing, it sets amount state.

https://stackoverflow.com/questions/55738134/should-i-include-setstate-in-usecallbacks-array-of-dependencies/55743744

Comment (danieledamiani) [reply packages/tempus-client/src/components/detail/advanced/detailMint.tsx:124]: good catch, please ignore my other comments

**Pull request merged**